Automattic/sensei
# Add Pricing side item for displating upgrade to sensei pro option

Fixes #4754

### Changes proposed in this Pull Request
- Add a new sidebar item for Pricing on Edit course page - design:  [Figma designs]
- <img width="301" alt="image" src="https://user-images.githubusercontent.com/7208249/154095688-24a452f9-0c7c-4b19-a2b5-3b85e2da9b67.png">

### Testing instructions
- Go to the Edit lesson page open editor settings: you should see Pricing metabox: 
<img width="290" alt="image" src="https://user-images.githubusercontent.com/7208249/153878243-d0def5fa-1e72-4b55-999f-104868719582.png">

When clicking on this link: 
<img width="206" alt="image" src="https://user-images.githubusercontent.com/7208249/153878311-34022f2f-751e-44fe-bacb-ea189372627f.png">
You should be redirected to : [Pricing](https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=course_pricing)

### New/Updated Hooks
Added hooks: 
1. **_senseiCoursePricingHide_**  - use it to hide pricing sidebar
2. **senseiCoursePricingDescription** : you can use it to pass an object like this
```  
{
		text: __(
			'Sell this course using WooCommerce - integrates with subscriptions, memberships, affiliates, and more.'
		),
		url:
			'https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=course_pricing',
	}
